src/ImageSharp/Formats/Tiff/TiffFrameMetadata.cs

```diff
@@ -77,7 +77,12 @@ internal static void Parse(TiffFrameMetadata meta, ExifProfile profile)
         {
             if (profile != null)
             {
-                meta.BitsPerSample = profile.GetValue(ExifTag.BitsPerSample) != null ? TiffBitsPerSample.GetBitsPerSample(profile.GetValue(ExifTag.BitsPerSample)?.Value) : null;
+                ushort[] bitsPerSampleValue = profile.GetValue(ExifTag.BitsPerSample)?.Value;
+                if (bitsPerSampleValue != null && TiffBitsPerSample.TryParse(bitsPerSampleValue, out TiffBitsPerSample bitsPerSample))
```

Comment:
This is 😍 though you don't need the null check since the struct already handles that.

Reply:
removed the null check